Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -63,20 +81,26 @@
     outputDirRoot=outputDir,
     rnd=rnd,
 )
+
 addSeeding(
     s,
     trackingGeometry,
     field,
+    TruthSeedRanges(pt=(1.0 * u.GeV, None), eta=(-4.0, 4.0), nHits=(9, None))
```

Comment:
```suggestion
    TruthSeedRanges(pt=(1.0 * u.GeV, None), eta=(-3.0, 3.0), nHits=(9, None))
```
Also, how many hits should we normally expect for the ODD?

Reply:
hmm good point I will just revert to the default here. but the ODD has 9 layers if I am not mistaken